src/tox_current_env/hooks4.py

```diff
@@ -209,6 +209,12 @@ def prepend_env_var_path(self):
     def runs_on_platform(self):
         return sys.platform
 
+    @classmethod
+    def python_spec_for_path(cls, path):
+        # https://github.com/fedora-python/tox-current-env/issues/77
+        ver, is_64 = sys.version_info, sys.maxsize != 2**32
+        return PythonSpec.from_string_spec(f"{sys.implementation}{ver.major}{ver.minor}-{is_64}")
```

Comment:
I'm not sure about the last part. The regex for parsing this is:

```python
PATTERN = re.compile(r"^(?P<impl>[a-zA-Z]+)?(?P<version>[0-9.]+)?(?:-(?P<arch>32|64))?$")
```

So I think that `is_64` should contain 32/64 and not a boolean value.

Reply:
I copy-pasted this from tox itself. If it's wrong, so is tox :(

In https://github.com/tox-dev/tox/pull/3325 this was added:

https://github.com/tox-dev/tox/blob/dde496464b25484720ea4327429ae17cafcf1131/src/tox/tox_env/python/api.py#L175-L176

```python
                    if str(spec_base.path) == sys.executable:
                        ver, is_64 = sys.version_info, sys.maxsize != 2**32
                        spec_base = PythonSpec.from_string_spec(f"{sys.implementation}{ver.major}{ver.minor}-{is_64}")
```